coralproject/talk
# [CORL-3041]: 8.7.1 QA bugfixes

## What does this PR do?

These changes update copy, ensure that rejection reason works with specific sites ban reject all comments, updates top commenter star styles on mobile, and ups our DSA additional comment URL max length.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags?
no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

Check that copy is updated for top commenter star tooltip, ban modal rejection reason in the admin detailed explanation label, and the `You must sign in to report a comment` text shown in the report form for anonymous DSA reports.

Check out the top commenter star with long and short usernames on mobile and see that in all cases, the star (plus any badges, etc.) are on a new line and aren't floating weirdly on the line with the username.

Single site ban a user in the admin and reject all of their comments. See that they are banned and the comments have the selected rejection reason shown when you check out their details in the moderation queue in the admin.

See that additional DSA urls in the DSA report form now have a max validation length of 1000 characters.

## Were any tests migrated to React Testing Library?



## How do we deploy this PR?
